kinto/tests/test_default_bucket.py

```diff
@@ -56,9 +56,10 @@ def test_second_call_on_default_bucket_collection_doesnt_raise_a_412(self):
         self.app.get(self.collection_url, headers=self.headers)
 
     def test_querystring_parameters_are_taken_into_account(self):
-        self.app.get(self.collection_url + '/records?_since=invalid',
-                     headers=self.headers,
-                     status=400)
+        resp = self.app.get(self.collection_url + '/records?_since=invalid',
+                            headers=self.headers,
+                            status=400)
+        self.assertIn('Content-Length', resp.headers)
```

Comment:
I wonder if we should create new tests for header checks; here for instance we're mixing querystring and headers. Thoughts?